src/Components/Authentication/Mobile/LoginForm.tsx

```diff
@@ -111,10 +103,7 @@ export const MobileLoginForm: FormComponentType = props => {
               <MobileHeader>Log in to Artsy</MobileHeader>
               {currentStep}
               {status && !status.success && <Error show>{status.error}</Error>}
-              <SubmitButton
-                disabled={!wizard.shouldAllowNext}
-                onClick={handleSubmit as any}
-              >
+              <SubmitButton onClick={handleSubmit as any}>
```

Comment:
So we leaving the button enabled at all times?

Reply:
Yup exactly, it won't allow going to the next page unless the current step/form is valid though